### PR TITLE
Bugfix/UIButtons

### PR DESCRIPTION
**Pull request by m4rtint**
- Credits page remove select button
- Forward button resets on end game panel

### CODE REVIEW

File: Assets/_PartyParkade/Scenes/DoozyMainMenu.unity [hunk 1]
```diff
@@ -12882,7 +12885,8 @@ MonoBehaviour:
       Event:
         m_PersistentCalls:
           m_Calls: []
-      GameEvents: []
+      GameEvents:
```
Comment (m4rtint) [inline Assets/_PartyParkade/Scenes/DoozyMainMenu.unity:12888]: All these edits are on the UIView Game Object.

When specific UIViews are shown, then the forward/backward button would show.

**Comment by m4rtint** [comment]
[Select Button Disappears when pressing "Enter" in Player Selection Screen](https://trello.com/c/sXt565X5/781-select-button-disappears-when-pressing-enter-in-player-selection-screen)

**Comment by m4rtint** [comment]
[Remove select button on the credit page](https://trello.com/c/2pP1PxI8/798-remove-select-button-on-the-credit-page)

**Comment by mikeslgoh** [comment]
[Deformable object missing reference](https://trello.com/c/8Nd9sQgT/806-deformable-object-missing-reference)

File: Assets/_PartyParkade/Script/PowerUp/Deformable.cs [hunk 2]
```diff
@@ -50,6 +47,11 @@ private void OnEnable()
         BaseWorldManager.OnFixedUpdate += DeformableUpdate;
     }
 
+    private void OnDisable()
+    {
+        BaseWorldManager.OnFixedUpdate -= DeformableUpdate;
```
Comment (mikeslgoh) [inline Assets/_PartyParkade/Script/PowerUp/Deformable.cs:52]: Is this the fix for the missing object reference?

Comment (m4rtint) [reply Assets/_PartyParkade/Script/PowerUp/Deformable.cs:52]: hopefully yeah

**Pull request merged**